instrumentation/aws-sdk/aws-sdk-2.2/library/src/main/java/io/opentelemetry/instrumentation/awssdk/v2_2/internal/BedrockRuntimeImpl.java

```diff
@@ -534,6 +578,30 @@ private static Long getUsageInputTokens(
         return null;
       }
       count = usage.asMap().get("input_tokens");
+    } else if (modelId.startsWith("meta.llama")) {
+      count = body.asMap().get("prompt_token_count");
+    } else if (modelId.startsWith("cohere.command-r")) {
+      // approximate input tokens based on prompt length
+      Document requestBody = executionAttributes.getAttribute(INVOKE_MODEL_REQUEST_BODY);
+      if (requestBody == null || !requestBody.isMap()) {
+        return null;
+      }
+      String prompt = requestBody.asMap().get("message").asString();
+      if (prompt == null) {
+        return null;
+      }
+      count = Document.fromNumber(Math.ceil(prompt.length() / 6.0));
```

Comment:
for my own understanding, is the use of dividing characters by "6" a fairly standard way of approximating average word length? It's used several times, perhaps it would make sense to make it a constant with a self-documenting name like `AVG_WORD_LENGTH` or something?

Reply:
It seems like 6 chars per token is the rate Bedrock uses to estimate token count for pricing, per https://docs.aws.amazon.com/bedrock/latest/userguide/model-customization-prepare.html. I will replace with a constant variable and add some comments in code.